src/app/users/view-user/view-user.component.html

```diff
@@ -45,16 +45,13 @@
           {{ userData.lastname }}
         </div>
 
-        <div *ngIf="userData.email">
 
-          <div fxFlex="50%" class="mat-body-strong">
-            Email
-          </div>
-
-          <div fxFlex="50%">
-            {{ userData.email }}
-          </div>
+        <div fxFlex="50%" class="mat-body-strong" *ngIf="userData.email">
+          Email
+        </div>
 
+        <div fxFlex="50%" *ngIf="userData.email">
+          {{ userData.email }}
         </div>
```

Comment:
@karantakalkar I have found removing this div, makes the entire thing working without any additional CSS changes. I agree this might not be the best method to achieve this, but in my opinion, the ngif is added for backward compatibility and while trying to create a user, I had to put an email as it was required. So, in this case, adding CSS to just maintain backward compatibility would be bad. So, I chose this approach. Correct me if I am wrong or if you don't like this approach, and I would rectify it accordingly.